docs/blog/2025-04-09-exploring-nx-console-with-mcp.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Vibe Exploring a Codebase with LLMs'
+slug: vibe-exploring-a-codebase
+authors: ['Juri Strumpflohner']
+tags: ['nx', 'nx-console', 'ai']
+cover_image: /blog/images/articles/bg-vibe-querying-codebase.jpg
```

Comment:
Make sure you are creating an avif for the cover image instead. Both a avif and a png/jpg are needed